mips/malta.c

```diff
@@ -181,6 +182,7 @@ void platform_init(int argc, char **argv, char **envp, unsigned memsize) {
   kmem_bootstrap();
   sleepq_init();
   thread_bootstrap();
+  pipe_init();
```

Comment:
We have not been using manual subsystem initialization since [SYSINIT](https://github.com/cahirwpz/mimiker/blob/69c98dee7682879b0aa6fdc0e7745b82eefa57e4/include/sysinit.h) was introduced.